node/silkworm/db/tables.hpp

```diff
@@ -57,37 +67,123 @@ inline constexpr db::MapConfig kCallToIndex{"CallToIndex"};
 inline constexpr db::MapConfig kCallTraceSet{"CallTraceSet", mdbx::key_mode::usual, mdbx::value_mode::multi};
 inline constexpr db::MapConfig kCode{"Code"};
 inline constexpr db::MapConfig kConfig{"Config"};
-inline constexpr db::MapConfig kContractCode{"HashedCodeHash"};
 inline constexpr db::MapConfig kDatabaseInfo{"DbInfo"};
 inline constexpr db::MapConfig kBlockTransactions{"BlockTransaction"};
+
+//! \details Store "current" state for accounts with hashed address key
+//! \remarks This table stores the same values for PlainState (Account record type) but with hashed key
+//! \struct
+//! \verbatim
+//!   key   : account address hash (20 bytes)
+//!   value : account encoded for storage
+//! \endverbatim
 inline constexpr db::MapConfig kHashedAccounts{"HashedAccount"};
+
+//! \details Store contract code hash for given contract by key hashed address + incarnation
+//! \remarks This table stores the same values for PlainCodeHash but with hashed key address
+//! \def "Incarnation" how many times given account was SelfDestruct'ed.
+//! \struct
+//! \verbatim
+//!   key   : contract address hash (32 bytes) + incarnation (u64 BE)
+//!   value : code hash (32 bytes)
+//! \endverbatim
+inline constexpr db::MapConfig kHashedCodeHash{"HashedCodeHash"};
+
+//! \details Store "current" state for contract storage with hashed address
+//! \remarks This table stores the same values for PlainState (storage record type) but with hashed key
+//! \struct
+//! \verbatim
+//!   key   : contract address hash (32 bytes) + incarnation (u64 BE)
+//!   value : storage key hash (32 bytes) + storage value (hash 32 bytes)
+//! \endverbatim
 inline constexpr db::MapConfig kHashedStorage{"HashedStorage", mdbx::key_mode::usual, mdbx::value_mode::multi};
 inline constexpr db::MapConfig kHeadBlock{"LastBlock"};
 inline constexpr db::MapConfig kHeadHeader{"LastHeader"};
 inline constexpr db::MapConfig kHeaderNumbers{"HeaderNumber"};
 inline constexpr db::MapConfig kHeadersSnapshotInfo{"HeadersSnapshotInfo"};
+
+//! \details Stores the last incarnation of last contract SelfDestruct
+//! \struct
+//! \verbatim
+//!   key   : contract address (unhashed 20 bytes)
+//!   value : incarnation (u64 BE)
+//! \endverbatim
 inline constexpr db::MapConfig kIncarnationMap{"IncarnationMap"};
 inline constexpr db::MapConfig kLogAddressIndex{"LogAddressIndex"};
 inline constexpr db::MapConfig kLogTopicIndex{"LogTopicIndex"};
 inline constexpr db::MapConfig kLogs{"TransactionLog"};
 inline constexpr db::MapConfig kMigrations{"Migration"};
-inline constexpr db::MapConfig kPlainContractCode{"PlainCodeHash"};
+
+//! \details Store contract code hash for given contract address + incarnation
+//! \def "Incarnation" how many times given account was SelfDestruct'ed.
+//! \struct
+//! \verbatim
+//!   key   : contract address (20 bytes) + incarnation (u64 BE)
+//!   value : code hash (32 bytes)
+//! \endverbatim
+inline constexpr db::MapConfig kPlainContractHash{"PlainCodeHash"};
```

Comment:
Perhaps rename to `kPlainCodeHash` for consistency with the string and `kHashedCodeHash`.

Reply:
Yes.